sympy/stats/crv_types.py

```diff
@@ -1442,6 +1442,10 @@ def pdf(self, x):
         beta, mu = self.beta, self.mu
         return (1/beta)*exp(-((x-mu)/beta)+exp(-((x-mu)/beta)))
 
+    def cdf(self, x):
```

Comment:
The method name `_cdf` is used for the probability distributions.
```suggestion
    def _cdf(self, x):
```